pkg/ui/react-app/src/thanos/pages/blocks/helpers.test.tsx

```diff
@@ -656,3 +658,37 @@ describe('Filtered block pools', () => {
     expect(filteredBlockPoolArray[0].thanos.labels).toEqual(filteredBlocks[0].thanos.labels);
   });
 });
+
+const newBlockPools = blockPools;
```

Comment:
Any reason all of these variables aren't moved into the test itself?

Reply:
Ah! moved the constants in the test. thanks